### ISSUE TITLE
Update README with correct commands to get started.

### ISSUE THREAD

**Issue by mbruzek** [juju-solutions/bundle-cwr-ci#10]
I tried deploying cwr-ci in my environment but was unable to follow the instructions in the readme.

```
$ juju deploy cwr-ci
ERROR cannot resolve URL "cs:cwr-ci": charm or bundle not found
```
I don't know why this didn't work so I went to jujucharms.com and searched for cwr-ci and found the bundle name was cs:~juju-solutions/bundle/cwr-ci and that worked.

At the time I was not logged in to the charm store:
```
$ juju whoami
Controller:  localhost-localhost
Model:       default
User:        admin
$ charm whoami
not logged into https://api.jujucharms.com/charmstore
$ juju version
2.1-beta5-xenial-amd64
```
I have not tried it after logging in.

Please let me know if you need any other information

**Comment by mbruzek** [comment]
I hit this again today I had to use the namespace to deploy successfully.
```
juju deploy cs:~juju-solutions/cwr-ci
```

**Comment by kwmonroe** [comment]
fixed with #13
